packages/lesswrong/components/review/ReviewVotingPage.tsx

```diff
@@ -209,9 +209,9 @@ const styles = (theme: ThemeType): JssStyles => ({
   }
 });
 
-export type vote = {_id: string, postId: string, score: number, type?: string, reactions: string[]}
-export type quadraticVote = vote & {type: "quadratic"}
-export type qualitativeVote = vote & {type: "qualitative", score: 0|1|2|3|4}
+export type ReviewVote = {_id: string, postId: string, score: number, type?: string, reactions: string[]}
+export type quadraticVote = ReviewVote & {type: "quadratic"}
+export type qualitativeVote = ReviewVote & {type: "qualitative", score: 0|1|2|3|4}
```

Comment:
My recommendation is to make these PascalCase at the same time. (`F2`)

Reply:
I will get to this when dealing with the rest of the voting UI